inspirehep/modules/crossref/views.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""INSPIRE CrossRef blueprint."""
+
+from __future__ import absolute_import, division, print_function
+
+from flask import Blueprint, jsonify, request
+from flask_login import login_required
+
+from .core import get_json
+from .config  import response_code
```

Comment:
In general you don't want to do this, as it prevents this configuration from being overridden at a later point. If instead you do it like this https://github.com/inspirehep/inspire-next/blob/6bc729117b8b5721736c150341c78870392c87a4/inspirehep/modules/hal/ext.py#L41-L44
then you'll find that you can modify things in the `current_app.config` any time you want.